src/data_info/routes/path_lists.clj

```diff
@@ -15,7 +15,8 @@
           :middleware [otel-middleware]
           :query [params PathListSaveQueryParams]
           :body [body (describe Paths "The folder or file paths to process for the HT Path List file contents.")]
-          :responses {200 {:schema      FileStat
+          :responses {200 {:schema      {:file (describe FilteredStatInfo "File info")
```

Comment:
Looks like `FilteredStatInfo` comes from `common-swagger-api.schema.stats`, but I don't see that ns referenced in this file 🤔

Reply:
Good point. looks like `FileStat` came via `data-info.routes.schemas.stats`, so I set it up (now, after your review) so `FilteredStatInfo` can come in the same way